.github/workflows/pull-request.yml

```diff
@@ -106,12 +106,40 @@ jobs:
         run: mkdocs build
         working-directory: ./site
 
-  lint-helm:
-    name: Lint Helm chart
+  helm-testing:
+    name: Lint & Test Helm chart
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
-      - name: Lint Helm
-        uses: WyriHaximus/github-action-helm3@v2
+      - name: Checkout
+        uses: actions/checkout@v2
         with:
-          exec: helm lint ./helm/nessie/
+          fetch-depth: 0
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.8.1
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@main
+
+      - name: Run chart-testing (list-changed)
+        id: list-changed
+        run: |
+          changed=$(ct list-changed --target-branch ${{ github.event.repository.default_branch }})
+          if [[ -n "$changed" ]]; then
+            echo "::set-output name=changed::true"
```

Comment:
I was initially thinking that we'd run the helm tests only if there are any changes, but for now maybe let's just run them every time for a while just to get a feel for the stability of those tests